### ISSUE TITLE
[JIT][meta] Hybrid front-end robustness example-level testing

### ISSUE THREAD

**Issue by jamesr66a** [pytorch/pytorch#8452]
E2E tests: https://github.com/pytorch/pytorch/pull/8451

Working models:

- [x] DCGAN example
- [ ] Fast Neural Style Transfer
- [ ] MNIST (partially working)
- [x] Reinforcement learning policy network
- [ ] SNLI (runs with external Python calls)
- [x] Super-resolution network
- [ ] Time-sequence prediction (example needs to be written with script)
- [ ] VAE (only runs under eval)

Task tracking:

- [ONNX InstanceNormalization symbolic interferes with regular tracing](https://github.com/pytorch/pytorch/issues/8439)
- [Better support tensor constructors (e.g. zeros, ones) under tracing](https://github.com/pytorch/pytorch/issues/8443)
- [ArgumentStash error with mnist example under training](https://github.com/pytorch/pytorch/issues/8448)
- [LSTM in the JIT] (https://github.com/pytorch/pytorch/issues/8449)
- [Dropout in the JIT](https://github.com/pytorch/pytorch/issues/8450)

**Comment by wanchaol** [comment]
Track some potential issues when scripting Time-sequence prediction:
[#8724 Don't support default values for arguments in the script](https://github.com/pytorch/pytorch/issues/8724)
[#8751 Str aren't supported in print](https://github.com/pytorch/pytorch/issues/8751)
[#8769 IntList arguments for call are not valid](https://github.com/pytorch/pytorch/issues/8769)
[#8771 Don't support varargs in script](https://github.com/pytorch/pytorch/issues/8771)
